2021/1xxx/CVE-2021-1078.json

```diff
@@ -1,18 +1,84 @@
 {
```

Comment:
'https://nvidia.custhelp.com/app/answers/detail/a_id/5172 ' does not match '^(ftp|http)s?://\\S+$'